django/core/exceptions.py

```diff
@@ -71,6 +71,11 @@ class ImproperlyConfigured(Exception):
     pass
 
 
+class FilePermissionError(Exception):
```

Comment:
Python 3.3+ has `PermissionError` https://docs.python.org/3/library/exceptions.html#PermissionError We can use it directly, or subclass it. On Python 2 and Python 3.2, `FilePermissionError` can be a subclass of `OSError`.